programs/vote/src/vote_state/mod.rs

```diff
@@ -1162,9 +1164,25 @@ pub fn withdraw<S: std::hash::BuildHasher>(
         .checked_sub(lamports)
         .ok_or(InstructionError::InsufficientFunds)?;
 
+    let received_credits_previous_epoch = if vote_state.epoch_credits.is_empty() {
+        false
```

Comment:
@CriesofCarrots I'm unsure of this logic.

If vote_state.epoch_credits.is_empty() does that necessarily mean the vote account didn't receive credits in the previous epoch? Does this state get stored on chain and shared among validators? Is it possible for a newly-started validator to have incomplete information here?